config/config.php

```diff
@@ -14,7 +14,7 @@
  * @since         1.1.11
  * @license       https://opensource.org/licenses/mit-license.php MIT License
  */
-$versionFile = file(CORE_PATH . 'VERSION.txt');
+$versionFile = file(dirname(__DIR__) . '/VERSION.txt');
```

Comment:
I was thinking about this problem some more, and will put up a pull request to make `Configure::version()` better.

Reply:
It would be kind of nice to have a build in version command that you could just configure ... but it doesn't make or break Console imo.